webui/module/Auth/src/Auth/Controller/AuthController.php

```diff
@@ -253,7 +259,7 @@ public function getDirectorModel()
     *
     * @return object
     */
-   private function createNewLoginForm($form, $err_msg = null, $bsock = null)
+   private function createNewLoginForm($form, $multi_dird_env = null, $err_msg = null, $bsock = null)
```

Comment:
```suggestion
   private function createNewLoginForm($form, $multi_dird_env, $err_msg = null, $bsock = null)
```

Reply:
It is passed to the View and checked there, I would like to leave it as it is.